### ISSUE TITLE
Javascript inside IE comments does not obey defer_javascript filter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pagespeed/mod_pagespeed#602]
```
Using the latest development version of mod_pagespeed, if you place some 
javascript within an IE comment block, it is ignored by the defer_javascript 
filter.

This can cause issues where it is reliant on various javascripts having been 
loaded previously.

If the code is not within an IE comment block, it gets deferred as you would 
expect and there are no problems.


```

Original issue reported on code.google.com by `rwap.services` on 8 Jan 2013 at 10:27
- Blocked on: #553
